### PR TITLE
feature/7853-Chika-RemoveCovid19UpdatesFromHomeScreen

### PR DESCRIPTION
**Pull request by cadibemma**
## Description of Change

- Removed Covid-19 updates from the home screen

#### Before / After: 
&nbsp;&nbsp;

## Testing

- [ ] Tested on iOS 
- [ ] Tested on Android 

## Reviewer Validations

Covid-19 updates are removed from the home screen

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [ ] PR is connected to issue(s)
- [ ] Tests are included to cover this change (when possible)
- [ ] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [ ] No secrets or API keys are checked in
- [ ] All imports are absolute (no relative imports)
- [ ] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/screens/HomeScreen/HomeScreen.tsx [hunk 1]
```diff
@@ -61,7 +60,7 @@ import SandboxScreen from './ProfileScreen/SettingsScreen/DeveloperScreen/Sandbo
 import ManageYourAccount from './ProfileScreen/SettingsScreen/ManageYourAccount/ManageYourAccount'
 import NotificationsSettingsScreen from './ProfileScreen/SettingsScreen/NotificationsSettingsScreen/NotificationsSettingsScreen'
 
-const { WEBVIEW_URL_CORONA_FAQ, WEBVIEW_URL_FACILITY_LOCATOR } = getEnv()
```
Comment (Sparowhawk) [inline VAMobile/src/screens/HomeScreen/HomeScreen.tsx:64]: do we need to also clean up this URL from the environment variables?

Comment (theodur) [reply VAMobile/src/screens/HomeScreen/HomeScreen.tsx:64]: The Covid 19 URL is also used on the HealthScreen, so we'll still need it

**Review by theodur — APPROVED** [review]

**Review by dumathane — APPROVED** [review]

**Comment by TKDickson** [comment]
@cadibemma @theodur @dumathane heads up that I've updated the detox test for the home screen so it's no longer broken by this PR. It doesn't look like it's invalidated any of the PR review, but at least one of you should also take a look at those changes.

**Review by TKDickson — APPROVED** [review]
Good to merge to XL

**Pull request merged**